maven-plugin-annotations/src/main/java/org/apache/maven/plugins/annotations/Execute.java

```diff
@@ -45,7 +45,8 @@
     LifecyclePhase phase() default LifecyclePhase.NONE;
 
     /**
-     * goal to fork. Note that specifying a phase overrides specifying a goal.
+     * goal to fork. Note that specifying a phase overrides specifying a goal. The goal must be a goal of the same
```

Comment:
> ...the goal must be a goal
Looks a bit confusing. What I think you want to say is:
`The specified <code>goal</code> must be another existing goal of the same plugin`
(and yes, the code-tags are intended for javadoc)

Reply:
Yes, that's absolutely an improvement. Thanks for the suggestion!